compiler/semstmts.nim

```diff
@@ -583,6 +584,57 @@ proc globalVarInitCheck(c: PContext, n: PNode) =
   if n.isLocalVarSym or n.kind in nkCallKinds and usesLocalVar(n):
     localError(c.config, n.info, errCannotAssignToGlobal)
 
+proc makeVarTupleSection(c: PContext, n, a, def: PNode, typ: PType, symkind: TSymKind, origResult: var PNode): PNode =
```

Comment:
What is all of this code doing? What does it have to do with warning on overloaded copy?

Reply:
Sorry, that was probably merging conflicts leaking into the code.